activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/MessageTestConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package org.activiti.spring.boot;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.activiti.api.process.model.events.BPMNMessageEvent;
+import org.activiti.api.process.model.events.BPMNMessageReceivedEvent;
+import org.activiti.api.process.model.events.BPMNMessageSentEvent;
+import org.activiti.api.process.model.events.BPMNMessageWaitingEvent;
+import org.activiti.api.process.runtime.events.listener.BPMNElementEventListener;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MessageTestConfiguration {
+
+    public static Set<BPMNMessageEvent> messageEvents = new HashSet<>();
```

Comment:
This should be list to preserve order and distinct events.

Reply:
Set replaced with a list